src/llmcompressor/transformers/finetune/data/cnn_dailymail.py

```diff
@@ -1,20 +1,11 @@
-# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
```

Comment:
I think before we start removing all these, we should double check with @markurtz

Reply:
I'll remove the  the copyright diffs from this PR and we can land them later [here](https://github.com/vllm-project/llm-compressor/pull/992)